### DISCUSSION TITLE
assembly factory

### DISCUSSION THREAD

**Discussion by litelmedox** [HbmMods/Hbm-s-Nuclear-Tech-GIT#575] [General]
there only is a chemistry factory now mabey a assembly factory

**Comment by HbmMods** [comment]
there will be one some day, probably

**Comment by HbmMods** [comment]
![image](https://user-images.githubusercontent.com/16674258/167314774-3b6078ab-a573-44fd-871d-3c025fbf0c7c.png)

**Reply by ncr-t** [reply]
Sex.

**Reply by sdddddf80** [reply]
Hope this can replace the old factories which can just be used as furnace.

**Reply by ncr-t** [reply]
It's not replacing the old factories, it's a completely different machine. It's like the chemical factory but for assembly machines.

**Reply by litelmedox** [reply]
> ![image](https://user-images.githubusercontent.com/16674258/167314774-3b6078ab-a573-44fd-871d-3c025fbf0c7c.png)

YAY
